raiden_libs/messages/balance_proof.py

```diff
@@ -140,10 +140,11 @@ def balance_hash(self, value) -> None:
 
     @property
     def signer(self) -> str:
-        return eth_verify(
+        signer = eth_verify(
             decode_hex(self.signature),
             self.serialize_bin()
         )
+        return to_checksum_address(signer)
```

Comment:
I ran into the same problem before, can you make these changes to the other messages as well.

Reply:
yup, updated fee info and path messages too